### PR TITLE
fix dangling links

### PR DESCRIPTION
**Pull request by shwstppr**
Fixes fangling links as reported in https://github.com/apache/cloudstack/issues/4594

### CODE REVIEW

**Comment by shwstppr** [comment]
requesting docbuild

**Review by sureshanaparti — APPROVED** [review]
LGTM

**Comment by shwstppr** [comment]
requesting docbuild

**Comment by vladimirpetrov** [comment]
Links in the "Networks" page are still missing. Go to the page:

[https://acs-www.shapeblue.com/docs/WIP-PROOFING/pr184//adminguide/networking.html#about-virtual-networks](https://acs-www.shapeblue.com/docs/WIP-PROOFING/pr184//adminguide/networking.html#about-virtual-networks)

Try the following links:
[https://acs-www.shapeblue.com/docs/WIP-PROOFING/pr184//adminguide/advanced_zone_config.html#configure-guest-traffic-in-an-advanced-zone](https://acs-www.shapeblue.com/docs/WIP-PROOFING/pr184//adminguide/advanced_zone_config.html#configure-guest-traffic-in-an-advanced-zone)
image

[https://acs-www.shapeblue.com/docs/WIP-PROOFING/pr184//adminguide/advanced_zone_config.html#configuring-a-shared-guest-network](https://acs-www.shapeblue.com/docs/WIP-PROOFING/pr184//adminguide/advanced_zone_config.html#configuring-a-shared-guest-network)
image

[https://acs-www.shapeblue.com/docs/WIP-PROOFING/pr184//adminguide/advanced_zone_config.html#persistent-networks](https://acs-www.shapeblue.com/docs/WIP-PROOFING/pr184//adminguide/advanced_zone_config.html#persistent-networks)
image

[https://acs-www.shapeblue.com/docs/WIP-PROOFING/pr184//adminguide/advanced_zone_config.html#about-virtual-private-clouds](https://acs-www.shapeblue.com/docs/WIP-PROOFING/pr184//adminguide/advanced_zone_config.html#about-virtual-private-clouds)
image

[https://acs-www.shapeblue.com/docs/WIP-PROOFING/pr184//adminguide/advanced_zone_config.html#dns-and-dhcp](https://acs-www.shapeblue.com/docs/WIP-PROOFING/pr184//adminguide/advanced_zone_config.html#dns-and-dhcp)
image

[https://acs-www.shapeblue.com/docs/WIP-PROOFING/pr184//adminguide/advanced_zone_config.html#remote-access-vpn](https://acs-www.shapeblue.com/docs/WIP-PROOFING/pr184//adminguide/advanced_zone_config.html#remote-access-vpn)
image

[https://acs-www.shapeblue.com/docs/WIP-PROOFING/pr184//adminguide/advanced_zone_config.html#configuring-network-access-control-list](https://acs-www.shapeblue.com/docs/WIP-PROOFING/pr184//adminguide/advanced_zone_config.html#configuring-network-access-control-list)
image

[https://acs-www.shapeblue.com/docs/WIP-PROOFING/pr184//adminguide/advanced_zone_config.html#adding-a-security-group](https://acs-www.shapeblue.com/docs/WIP-PROOFING/pr184//adminguide/advanced_zone_config.html#adding-a-security-group)
image

[https://acs-www.shapeblue.com/docs/WIP-PROOFING/pr184//adminguide/advanced_zone_config.html#about-elastic-ip](https://acs-www.shapeblue.com/docs/WIP-PROOFING/pr184//adminguide/advanced_zone_config.html#about-elastic-ip)
image

All of them generate 'Not Found' error pages.

File: source/adminguide/virtual_machines.rst [hunk 1]
```diff
@@ -58,7 +58,7 @@ IP and the guest VM’s private IP. The VM’s original IP address is then
 released and returned to the pool of available public IPs. Optionally,
 you can also decide not to allocate a public IP to a VM in an
 EIP-enabled Basic zone. For more information on Elastic IP, see
-`“About Elastic IP” <networking2.html#about-elastic-ip>`_.
+`“About Elastic IP” <advanced_zone_config.html#about-elastic-ip>`_.
```
Comment (DaanHoogland) [inline source/adminguide/virtual_machines.rst:61]:
```suggestion
`“About Elastic IP” <networking/elastic_ips.html>`_.
```

File: source/adminguide/networking.rst [hunk 2]
```diff
@@ -60,7 +60,7 @@ account. Isolated networks have the following properties.
    entire network
 
 For more information, see `“Configure Guest Traffic in an Advanced Zone” 
-<networking2.html#configure-guest-traffic-in-an-advanced-zone>`_.
+<advanced_zone_config.html#configure-guest-traffic-in-an-advanced-zone>`_.
```
Comment (DaanHoogland) [inline source/adminguide/networking.rst:63]:
```suggestion
<networking_and_traffic.html#configure-guest-traffic-in-an-advanced-zone>`_.
```

File: source/adminguide/networking.rst [hunk 3]
```diff
@@ -85,7 +85,7 @@ Basic zones in CloudStack 3.0.3 and later versions.
 -  Source NAT per zone is not supported in Shared Network when the
    service provider is virtual router. However, Source NAT per account
    is supported. For information, see `“Configuring a Shared Guest 
-   Network” <networking2.html#configuring-a-shared-guest-network>`_.
+   Network” <advanced_zone_config.html#configuring-a-shared-guest-network>`_.
```
Comment (DaanHoogland) [inline source/adminguide/networking.rst:88]:
```suggestion
   Network” <networking_and_traffic.html#configuring-a-shared-guest-network>`_.
```

File: source/adminguide/networking.rst [hunk 4]
```diff
@@ -271,7 +271,7 @@ To create a network offering:
       or not. The network that you can provision without having to
       deploy a VM on it is termed persistent network. For more
       information, see `“Persistent
-      Networks” <networking2.html#persistent-networks>`_.
+      Networks” <advanced_zone_config.html#persistent-networks>`_.
```
Comment (DaanHoogland) [inline source/adminguide/networking.rst:274]:
```suggestion
      Networks” <networking_and_traffic.html#persistent-networks>`_.
```

File: source/adminguide/networking.rst [hunk 5]
```diff
@@ -284,7 +284,7 @@ To create a network offering:
       isolated part of CloudStack. A VPC can have its own virtual
       network topology that resembles a traditional physical network.
       For more information on VPCs, see `“About Virtual
-      Private Clouds” <networking2.html#about-virtual-private-clouds>`_.
+      Private Clouds” <advanced_zone_config.html#about-virtual-private-clouds>`_.
```
Comment (DaanHoogland) [inline source/adminguide/networking.rst:287]:
```suggestion
      Private Clouds” <networking_and_traffic.html#about-virtual-private-clouds>`_.
```

File: source/adminguide/networking.rst [hunk 6]
```diff
@@ -302,8 +302,10 @@ To create a network offering:
       =================== ============================================================================ ============= =============
       Supported Services  Description                                                                  Isolated      Shared   
       =================== ============================================================================ ============= =============
-      DHCP                For more information, see `“DNS and DHCP” <networking2.html#dns-and-dhcp>`_. Supported     Supported
-      DNS                 For more information, see `“DNS and DHCP” <networking2.html#dns-and-dhcp>`_. Supported     Supported
+      DHCP                For more information, see `“DNS and                                          Supported     Supported
+                          DHCP” <advanced_zone_config.html#dns-and-dhcp>`_.
+      DNS                 For more information, see `“DNS and                                          Supported     Supported
+                          DHCP”  <advanced_zone_config.html#dns-and-dhcp>`_.
```
Comment (DaanHoogland) [inline source/adminguide/networking.rst:308]:
```suggestion
      DHCP                For more information, see `“DNS and                                          Supported     Supported
                          DHCP” <networking_and_traffic.html#dns-and-dhcp>`_.
      DNS                 For more information, see `“DNS and                                          Supported     Supported
                          DHCP”  <networking_and_traffic.html#dns-and-dhcp>`_.
```

File: source/adminguide/networking.rst [hunk 7]
```diff
@@ -318,13 +320,13 @@ To create a network offering:
                           virtual router or any other Port Forwarding providers that have
                           been configured in the cloud.
       VPN                 For more information, see `“Remote Access                                    Supported     Not Supported
-                          VPN” <networking2.html#remote-access-vpn>`_.
+                          VPN” <advanced_zone_config.html#remote-access-vpn>`_.
```
Comment (DaanHoogland) [inline source/adminguide/networking.rst:323]:
```suggestion
                          VPN” <networking_and_traffic.html#remote-access-vpn>`_.
```

File: source/adminguide/networking.rst [hunk 8]
```diff
@@ -318,13 +320,13 @@ To create a network offering:
...
       User Data           For more information, see `“User Data and Meta                               Not Supported Supported
                           Data” <api.html#user-data-and-meta-data>`_.
-      Network ACL         For more information, see `“Configuring Network Access                       Supported     Not Supported
-                          Control List” <networking2.html#configuring-network-access-control-list>`_.
+      Network ACL         For more information, see `“Configuring Network Access Control               Supported     Not Supported
+                          List” <advanced_zone_config.html#configuring-network-access-control-list>`_.
```
Comment (DaanHoogland) [inline source/adminguide/networking.rst:327]:
```suggestion
                          List” <networking_and_traffic.html#configuring-network-access-control-list>`_.
```

File: source/adminguide/networking.rst [hunk 9]
```diff
@@ -318,13 +320,13 @@ To create a network offering:
...
       Security Groups     For more information, see `“Adding a Security                                Not Supported Supported
-                          Group” <networking2.html#adding-a-security-group>`_.
+                          Group” <advanced_zone_config.html#adding-a-security-group>`_.
```
Comment (DaanHoogland) [inline source/adminguide/networking.rst:329]:
```suggestion
                          Group” <networking_and_traffic.html#adding-a-security-group>`_.
```

**Review by DaanHoogland — CHANGES_REQUESTED** [review]
I think i have all of @vladimirpetrov 's sangling links now :open_mouth:

**Comment by shwstppr** [comment]
requesting docbuild

File: source/adminguide/networking.rst [hunk 10]
```diff
@@ -318,13 +320,13 @@ To create a network offering:
...
       User Data           For more information, see `“User Data and Meta                               Not Supported Supported
                           Data” <api.html#user-data-and-meta-data>`_.
-      Network ACL         For more information, see `“Configuring Network Access                       Supported     Not Supported
-                          Control List” <networking2.html#configuring-network-access-control-list>`_.
+      Network ACL         For more information, see `“Configuring Network Access Control               Supported     Not Supported
+                          List” <networking_and_traffic.html#configuring-network-access-control-list>`_.
       Security Groups     For more information, see `“Adding a Security                                Not Supported Supported
-                          Group” <networking2.html#adding-a-security-group>`_.
+                          Group” <networking_and_traffic.html#adding-a-security-group>`_.
       =================== ============================================================================ ============= =============
```
Comment (DaanHoogland) [inline source/adminguide/networking.rst:330]: this table has dissappeared ???
image

**Comment by shwstppr** [comment]
requesting docbuild

File: source/adminguide/networking.rst (hunk 10 above)
Comment (shwstppr) [reply source/adminguide/networking.rst:330]: @DaanHoogland should be fixed now.

PS: I had to clear cache for the page to see changes

**Review by DaanHoogland — APPROVED** [review]

**Pull request merged**